app/_src/gateway/install/docker/build-custom-images.md

```diff
@@ -144,13 +144,105 @@ RUN set -ex; \
     && apk del .build-deps \
     && kong version
 
+{{ dockerfile_run_steps }}
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+{{ dockerfile | indent }}
+{% endif_version %}
+
+{% if_version gte:3.4.x %}
```

Comment:
❗ For reviewers: I feel like what I did here (made two different capture blocks to do conditional rendering) isn't the best solution, but I couldn't figure out how to just conditionally render the Alpine navtab. Is there a way to do that or a better way to do the conditional rendering here?

Reply:
@lena-larionova might know of one, but I can't think of a better way to do this than this.

Reply:
Just tried a bunch of different approaches and nothing else worked. Let's go with this.